build.sbt

```diff
@@ -2,12 +2,13 @@ organization := "org.zalando"
 
 name := "play-etcd-watcher"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.2"
```

Comment:
I think it is fine switching to 2.12 as proposed in the PR. For backward compatibility we'd need to create a patch and release branch anyways, and then merge the branch to both master and release branch. Once we need to patch an older version, we might think of some improvements in the build process.